src/Experimental/Template.fs

```diff
@@ -40,8 +40,8 @@ type Binder = Xml -> Xml
 /// Extract an XML/HTML template from the input 'xml', in 'ns1:keyhere' form
 /// for tags that can be parsed. Replace those tags with their values as defined
 /// in the 'bindings'.
-let extract_template (nsx : string) (bindings : Map<string,Binder>) (xml : Xml) =
-  let rec extract_template' (Xml(nodelist)) (Xml(result)) : Xml =
+let extractTemplate (nsx : string) (bindings : Map<string,Binder>) (xml : Xml) =
+  let rec extractTemplateAux (Xml(nodelist)) (Xml(result)) : Xml =
```

Comment:
Isn't `'` a good char to use for nested private functions? Not all non-english speakers understand `auxiliary`